wfapi/error.go

```diff
@@ -199,18 +192,20 @@ func ErrorFormulaInvalid(reason string) error {
 	)
 }
 
+// DEPRECATED: message adds no value
 // ErrorFormulaExecutionFailed is returned to wrap generic errors that cause
 // formula execution to fail.
 //
 // Errors:
 //
 //    - warpforge-error-formula-execution-failed --
 func ErrorFormulaExecutionFailed(cause error) error {
-	return serum.Errorf(ECodeFormulaExecutionFailed,
-		"formula execution failed: %w", cause,
+	return serum.Error(ECodeFormulaExecutionFailed, serum.WithCause(cause),
+		serum.WithMessageLiteral("formula execution failed"),
```

Comment:
Agree with this one being deprecated.  I think this one is an example of where having a "cause" parameter at all was probably a misstep.

Formula execution does not have _that_ many moving parts.  Maybe this error code exists, and maybe it has a constructor function, but the constructor(s?) should probably be functions accepting some specific details as parameters... not an opaque, arbitrary cause.